stylesheets/partials/core/_colors.scss

```diff
@@ -10,6 +10,10 @@ $tan: rgb(216, 210, 196);
 $tan-light: rgb(248,240,219);
 $purple-dark: rgb(53, 46, 79);  /* #352e4f; */
 $emergency-orange: #fa542d;
+$form-response-success-border: #93d493;
```

Comment:
Not that this would really probably matter, much, but would the success border be the same regardless if it's tied to a form?